Cactooz/StarWarsWiki
# Details view favorite buttons

### Additions
- Added “Add Favorite” and “Remove Favorite” buttons to detailsView with fitting functions
- Reformatted model.js
- Added check to see if user is logged in. If user is not logged in, the buttons will disappear

### Testing
1. Go to any details page without being logged in. Favorite buttons should not be visible
2. Sign in using the button in the header
3. Now add the page to your favorites, using the button under the image
4. Go to profile and see the newest addition to your list
5. Go back to the details page and remove from favorites
6. Check list again to see if removal was successful
